gradle-plugin/src/main/kotlin/dev/icerock/gradle/MultiplatformResourcesPlugin.kt

```diff
@@ -28,7 +28,7 @@ class MultiplatformResourcesPlugin : Plugin<Project> {
         val mrExtension =
             target.extensions.create<MultiplatformResourcesPluginExtension>("multiplatformResources")
 
-        mrExtension.onChange = {
+        target.afterEvaluate {
```

Comment:
why it was changed to `afterEvaluate`? it have reason?

Reply:
we want the callback to fire once all properties set on the extension, it gets awkward with the custom callback.  afaik this is the way to go, eg https://github.com/cashapp/sqldelight/blob/master/sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/SqlDelightPlugin.kt#L41